projects.go

```diff
@@ -1955,6 +1955,43 @@ func (s *ProjectsService) ChangeAllowedApprovers(pid interface{}, opt *ChangeAll
 	return pa, resp, err
 }
 
+// ProjectPullMirrorDetails represent the details of the configuration pull mirror and its update status
+//
+// GitLab API docs: https://docs.gitlab.com/ee/api/projects.html#get-a-projects-pull-mirror-details
+type ProjectPullMirrorDetails struct {
+	ID                     int        `json:"id"`
+	LastError              string     `json:"last_error"`
+	LastSuccessfulUpdateAt *time.Time `json:"last_successful_update_at"`
+	LastUpdateAt           *time.Time `json:"last_update_at"`
+	LastUpdateStartedAt    *time.Time `json:"last_update_started_at"`
+	UpdateStatus           string     `json:"update_status"`
+	URL                    string     `json:"url"`
+}
+
+// GetProjectPullMirrorDetails returns the pull mirror details
+//
+// GitLab API docs: https://docs.gitlab.com/ee/api/projects.html#get-a-projects-pull-mirror-details
+func (s *ProjectsService) GetProjectPullMirrorDetails(pid interface{}, options ...RequestOptionFunc) (*ProjectPullMirrorDetails, *Response, error) {
+	project, err := parseID(pid)
+	if err != nil {
+		return nil, nil, err
+	}
+	u := fmt.Sprintf("projects/%s/mirror/pull", PathEscape(project))
+
+	req, err := s.client.NewRequest(http.MethodGet, u, nil, options)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	model := new(ProjectPullMirrorDetails)
```

Comment:
`model` is kind of a weird name (IMHO)... Maybe just `pmd` to keep it more consistent with other namings?

Reply:
done